job_definitions/notify_suppliers_of_new_questions_answers.yml

```diff
@@ -0,0 +1,56 @@
+{% set environments = ['preview', 'production'] %}
+---
+{% for environment in environments %}
+- job:
+    name: "notify-suppliers-of-new-questions-answers-{{ environment }}"
+    display-name: "Notify suppliers of new questions and answers - {{ environment }}"
+    project-type: freestyle
+    description: "Send email notifications to supplier users about new questions and answers."
+    parameters:
+      - string:
+          name: SUPPLIER_IDS
+          description: "List of supplier IDs to be emailed e.g. 3,4,6"
+      - bool:
+          name: DRY_RUN
+          default: false
+          description: "List notifications that would be sent without sending the emails"
+    scm:
+      - git:
+          url: git@github.com:alphagov/digitalmarketplace-scripts.git
+          credentials-id: github_com_and_enterprise
+          branches:
+            - master
+          wipe-workspace: false
+{% if environment == 'production' %}
+    triggers:
+      - timed: "0 8 * * *"
+{% endif %}
+    publishers:
+      - trigger-parameterized-builds:
+          - project: notify-slack
+            condition: UNSTABLE_OR_WORSE
+            predefined-parameters: |
+              USERNAME=question-and-answers
+              JOB=Notify suppliers of new questions and answers {{ environment }}
+              ICON=:question:
+              STAGE={{ environment }}
+              STATUS=FAILED
+              URL=<${BUILD_URL}consoleFull|#${BUILD_NUMBER}>
+              CHANNEL=#dm-release
+    builders:
+      - shell: |
+          [ -d venv ] || virtualenv venv
+
+          . ./venv/bin/activate
+          pip install -r requirements.txt
+
+          if [ -n "SUPPLIER_IDS" ]; then
```

Comment:
Should `SUPPLIER_IDS` have a `$` in front of it here? (I don't know for sure, but suspect yes).